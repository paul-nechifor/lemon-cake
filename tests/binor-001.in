(| 1 2 4)
(| 0xff000000 0x0f0000ff)
(| 0xff00000000000000 0x0f0000ff000000ff)
