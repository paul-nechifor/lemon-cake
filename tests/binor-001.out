7
4278190335
-72056498821267201
